### ISSUE TITLE
Soft aggregation 

### ISSUE THREAD

**Issue by npmhung** [seoungwugoh/STM#7]
Hi,
When I checked your code with the supplementary, I found out that the way soft aggregation is calculated in the paper is different from your code. 

In the code, you apply softmax directly to the output of model and then perform logit function.
In the paper, you use logit and then softmax.

Is this an error or did you do that on purpose?
